doc/whats_new.rst

```diff
@@ -23,6 +23,8 @@ Changelog
 
     - Add label center of mass function :func:`mne.Label.center_of_mass` by `Eric Larson`_
 
+    - Added :func:`mne.viz.ica.plot_properties` that allows ploting of independent component properties similar to ``pop_prop`` in EEGLAB. Also :class:`mne.preprocessing.ica.ICA` has ``plot_properties`` method now. Added by `Mikołaj Magnuski`_
```

Comment:
Or maybe I'll just change the name to `plot_ica_properties` and worry about channel properties in the next PR?